modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/message/ErrorMessage.java

```diff
@@ -34,6 +34,8 @@ public interface ErrorMessage extends NetworkMessage, Serializable {
      */
     UUID queryId();
 
+    int executionToken();
```

Comment:
I would recommend adding javadoc for consistency with other methods.

Reply:
done, thanks